pkg/operator/ceph/object/spec.go

```diff
@@ -569,7 +570,7 @@ func addPortToEndpoint(endpoints *v1.Endpoints, name string, port int32) {
 }
 
 func getLabels(name, namespace string, includeNewLabels bool) map[string]string {
-	labels := controller.CephDaemonAppLabels(AppName, namespace, "rgw", name, includeNewLabels)
+	labels := controller.CephDaemonAppLabels(AppName, namespace, config.RgwType, name, "radosgw", instanceName(name), "cephobjectstores", includeNewLabels)
```

Comment:
Are we not doing the full `<kind>.ceph.rook.io`?

```suggestion
	labels := controller.CephDaemonAppLabels(AppName, namespace, config.RgwType, name, "radosgw", instanceName(name), "cephobjectstores.ceph.rook.io", includeNewLabels)
```